lib/Mojo/UserAgent.pm

```diff
@@ -16,6 +16,7 @@ use constant DEBUG => $ENV{MOJO_USERAGENT_DEBUG} || 0;
 
 has ca              => sub { $ENV{MOJO_CA_FILE} };
 has cert            => sub { $ENV{MOJO_CERT_FILE} };
+has verify          => sub { $ENV{MOJO_VERIFY} };
```

Comment:
I think the option should simply be called `insecure`, like in curl.

Reply:
And be a boolean attribute of course. With `-k/--insecure` flag in `Mojolicious::Command::get`.

Reply:
And it appears the actual attribute accessor does not have a test yet.